schema/deploy/computed_columns/form_result_requested_change_comments.sql

```diff
@@ -15,6 +15,7 @@ begin;
             where rc.application_id = form_result.application_id
             and rc.form_id = form_result.form_id
             and rc.comment_type = 'requested change'::ggircs_portal.review_comment_type
+            and rc.deleted_by is null
```

Comment:
Do this in `form_result_internal_general_comments.sql` as well

Reply:
Added